torchgeo/datasets/geo.py

```diff
@@ -461,8 +461,13 @@ def _merge_files(self, filepaths: Sequence[str], query: BoundingBox) -> Tensor:
             )
         else:
             dest, _ = rasterio.merge.merge(vrt_fhs, bounds, self.res)
-        dest = dest.astype(np.int32)
 
+        # fix numpy dtypes which are not supported by pytorch tensors
+        if dest.dtype == np.uint16:
+            dest = dest.astype(np.int32)
+        elif dest.dtype == np.uint32:
+            dest = dest.astype(np.int64)
+        
```

Comment:
Black/flake8 are complaining about this empty line with whitespace characters. You should be able to fix this by running `black .` if you have black installed.